apps/base-docs/docs/pages/wallet-app/mini-apps.mdx

```diff
@@ -0,0 +1,102 @@
+# Mini Apps
+
+We're so excited that mini apps are coming to Coinbase Wallet! The purpose of this guide is to go over how to build or update your mini app so it works as well as possible in Coinbase Wallet.
+
+## Using MiniKit
+
+If you use MiniKit and/or follow the [MiniKit quickstart guide](/builderkits/minikit/quickstart), your mini app will work out of the box in Coinbase Wallet!
+
+For reference, MiniKit is easiest way to build mini apps on Base, allowing developers to easily build mini apps without needing to know the details of the SDK implementation. It integrates seamlessly with OnchainKit components and provides Coinbase Wallet-specific hooks.
+
+If you're already using MiniKit and experiencing issues, you can refer to the [Debugging guide](/builderkits/minikit/debugging) and if your issue isn't covered there we'd greatly appreciate if you could flag it to our team.
+
+## Authentication
+
+As a general rule of thumb for building any mini app, we recommend that you do not automatically request that the user logs in/signs a message and instead that authentication is only used when it's needed(eg. signing up for an event, viewing authenticated resources, etc).
+Below we will quickly cover the different methods of authentication offered for mini apps and how well each of them work in Coinbase Wallet:
+
+### Wallet
+
+Because users in Coinbase Wallet have an in-app smart wallet that doesn't require any app switching, we recommend wallet authentication as the primary method of authentication for developers looking to create a persisted session for the mini app user.
+
+As described below we don't think it's the best practice to prompt the user to log in before that authentication will allow them to do something else in your mini app, but for cases where do you want a secure, persisted session, using a wallet connection is a great option.
+
+### Context Data
+
+All mini app host apps(including Coinbase Wallet) return [context data](https://miniapps.farcaster.xyz/docs/sdk/context), which tells developers about the app/mini app host the user is accessing their mini app in, as well as **which user** is interacting with their mini app.
+
+While this data can technically be spoofed, not only can you be sure that Coinbase Wallet will always give you the accurate data as a developer but also making use of this context data is super helpful because it doesn't require any other action from the user.
```

Comment:
if the data can be spoofed, how can they be sure that what we're giving is accurate? seems contradictory